test/controllers/checklists_controller_test.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+# Controller tests for checklists
+class ChecklistsControllerTest
+  #   -----------
+  #    checklist
+  #   -----------
+
+  # Prove that Life List goes to correct page which has correct content
+  def test_checklist_for_user
+    login
+    user = users(:rolf)
+    expect = Name.joins(observations: :user).
+             where(Observation[:user_id] == user.id).
+             where(Name[:rank] == Name.ranks[:Species]).distinct
+
+    get(checklist_path(user_id: user.id))
```

Comment:
@JoeCohen I think i was working on this about the same time you were writing... I reverted back to this.
The path helpers don't work in controller tests because we are overriding `get` and `post` with our own methods. (Or they may not work in `FunctionalTestCase` at all... but it's limited to controller tests, thankfully.)

Reply:
That makes perfect sense. (The helpers do work in contexts other than the overridden methods.)